ocaml/xapi/helpers.ml

```diff
@@ -1826,14 +1830,16 @@ end = struct
 
   (** protect ourselves from concurrent writes to files *)
   module Config : sig
-    val update : accept:string -> unit
+    val update : accept:string -> client_auth_name:string option -> unit
     (** create or update stunnel config file *)
   end = struct
     let m = Mutex.create ()
 
     let current_accept = ref None
 
-    let rewrite_xapi_ssl_config_file ~accept =
+    let current_client_auth_name = ref None
+
+    let rewrite_xapi_ssl_config_file ~accept ~client_auth_name =
```

Comment:
I think `rewrite` does not capture what this code is doing - it does not read the file and modifies it. It generates a configuration. So I would call it `emit_xapi_ssl_config` or `generate_...`.

Reply:
IMO "rewrite" is correct in the sense that it writes the file again to update its contents.

Reply:
Done.